src/cephadm/cephadm

```diff
@@ -2945,6 +2945,7 @@ def install_base_units(ctx, fsid):
     with open(ctx.unit_dir + '/ceph.target.new', 'w') as f:
         f.write('[Unit]\n'
                 'Description=All Ceph clusters and services\n'
+                'BindsTo=docker.service\n'
```

Comment:
please do this only, in case we're actually using docker.

Reply:
do u mean ceph may use podman as container engine, so add this only for using docker engine.

Reply:
exactly.